std/valgrind/callgrind.zig

```diff
@@ -0,0 +1,87 @@
+const std = @import("../index.zig");
+const valgrind = std.valgrind;
+
+pub const CallgrindClientRequest = extern enum {
+    DumpStats = valgrind.ToolBase("CT"),
+    ZeroStats,
+    ToggleCollect,
+    DumpStatsAt,
+    StartInstrumentation,
+    StopInstrumentation,
+};
+
+fn doCallgrindClientRequestExpr(default: usize, request: CallgrindClientRequest,
+        a1: usize, a2: usize, a3: usize, a4: usize, a5: usize
+    ) usize
+{
+    return valgrind.doClientRequest(
+        default,
+        @intCast(usize, @enumToInt(request)),
+        a1, a2, a3, a4, a5);
+}
+
+fn doCallgrindClientRequestStmt(request: CallgrindClientRequest,
+        a1: usize, a2: usize, a3: usize, a4: usize, a5: usize
+    ) void
+{
+    _ = doCallgrindClientRequestExpr(0, request, a1, a2, a3, a4, a5);
+}
+
+
+
+// Dump current state of cost centers, and zero them afterwards
+pub fn dumpStats() void {
+    doCallgrindClientRequestStmt(CallgrindClientRequest.DumpStats,
+        0, 0, 0, 0, 0);
+}
+
+
+// Dump current state of cost centers, and zero them afterwards.
```

Comment:
These should be [doc comments](https://ziglang.org/documentation/master/#Doc-comments)